src/libraries/System.Security.Cryptography.Pkcs/tests/SignatureSupport.cs

```diff
@@ -0,0 +1,14 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Xml;
+using Xunit;
+
+namespace System.Security.Cryptography.Pkcs.Tests
+{
+    public class SignatureSupport
+    {
+        public static bool SupportsRsaSha1Signatures { get; } =
+            System.Security.Cryptography.Tests.SignatureSupport.CanProduceSha1Signature(RSA.Create());
```

Comment:
To fix the .NET Framework build:

```suggestion
        public static bool SupportsRsaSha1Signatures { get; } =
#if NETFRAMEWORK
            true;
#else
            System.Security.Cryptography.Tests.SignatureSupport.CanProduceSha1Signature(RSA.Create());
#endif
```

Reply:
Actually, since you hit the same problem in your other PR, make it would just make sense to fix this in `SignatureSupport`. Something like:


```diff
diff --git a/src/libraries/Common/tests/System/Security/Cryptography/SignatureSupport.cs b/src/libraries/Common/tests/System/Security/Cryptography/SignatureSupport.cs
index abdbabb14b8..172e09ec63a 100644
--- a/src/libraries/Common/tests/System/Security/Cryptography/SignatureSupport.cs
+++ b/src/libraries/Common/tests/System/Security/Cryptography/SignatureSupport.cs
@@ -7,6 +7,10 @@ internal static class SignatureSupport
     {
         internal static bool CanProduceSha1Signature(AsymmetricAlgorithm algorithm)
         {
+#if NETFRAMEWORK
+            algorithm.Dispose();
+            return true;
+#else
             // We expect all non-Linux platforms to support SHA1 signatures, currently.
             if (!OperatingSystem.IsLinux())
             {
@@ -46,6 +50,7 @@ internal static bool CanProduceSha1Signature(AsymmetricAlgorithm algorithm)
                 default:
                     throw new NotSupportedException($"Algorithm type {algorithm.GetType()} is not supported.");
             }
+#endif
         }
     }
 }
```